contentgrid-appserver-query-engine-impl-jooq/src/main/java/com/contentgrid/appserver/query/engine/jooq/JOOQTableCreator.java

```diff
@@ -48,6 +102,29 @@ private void createTableForEntity(DSLContext dslContext, Entity entity) {
         } catch (BadSqlGrammarException e) {
             throw new InvalidSqlException(e.getMessage(), e);
         }
+
+        // Create FTS indices.
+        entity.getSearchFilters()
+                .stream()
+                .filter(searchFilter -> searchFilter instanceof AttributeSearchFilter && ((AttributeSearchFilter) searchFilter).getOperation().equals(AttributeSearchFilter.Operation.FTS))
+                .map(searchFilter -> application.resolvePropertyPath(entity, ((AttributeSearchFilter) searchFilter).getAttributePath()))
+                .distinct()
+                .forEach(simpleAttribute -> createFTSIndex(dslContext, entity, simpleAttribute));
+    }
+
+    @Allow.PlainSQL
+    private void createFTSIndex(@NonNull DSLContext dslContext, @NonNull Entity entity, @NonNull SimpleAttribute attribute) throws RuntimeException {
+        String tableName = entity.getTable().getValue();
+        String ftsColumnName = attribute.getColumn().getValue();
+        String indexName = "%s_%s_fts_idx".formatted(tableName, ftsColumnName);
+        Locale attributeLocale = attribute.getLocale();
+        if (!SUPPORTED_LOCALES.contains(attributeLocale)) throw new InvalidArgumentModelException("Locale (%s) is not supported for full-text search.".formatted(attributeLocale));
+
+        log.debug("Creating an FTS index ({}) on table ({}) for column ({}).", indexName, tableName, ftsColumnName);
+        // JOOQ does not seem to be flexible enough to create the FTS index with the required configuration, so we use a prepared statement.
+        // Allow.PlainSQL since this code is executed only during the model definition phase, not during request processing.
+        dslContext.execute(ftsIndexPreparedStatement, DSL.name(indexName), DSL.name(tableName),
+                DSL.inline(attributeLocale.getDisplayLanguage(ENGLISH)), DSL.inline(ftsColumnName));
```

Comment:
I think `ftsColumnName` needs to be a `DSL.name()`? It's a column name, not text.